ts/Core/Chart/Chart.ts

```diff
@@ -2445,30 +2443,39 @@ class Chart {
 
         // Use Math.max to prevent negative plotHeight
         chart.plotHeight = Math.max(chart.plotHeight - expectedSpace, 0);
-        const tempHeight = chart.plotHeight;
+        let tempWidth = chart.plotWidth,
+            tempHeight = chart.plotHeight;
 
         // Get margins by pre-rendering axes
         axes.forEach((axis): void => axis.setScale());
         chart.getAxisMargins();
 
         // If the plot area size has changed significantly, calculate tick
         // positions again
-        const redoHorizontal = tempWidth / chart.plotWidth > 1.1,
-            // Height is more sensitive, use lower threshold
-            redoVertical = tempHeight / chart.plotHeight > 1.05;
-
-        if (redoHorizontal || redoVertical) {
+        let redoHorizontal = tempWidth !== chart.plotWidth,
+            redoVertical = tempHeight !== chart.plotHeight,
```

Comment:
A notable change here is that previously we accepted a failure margin (> 1.1, > 1.05 respectively). I think we should keep that acceptance margin on these lines, but if the `overheatProtector` is > 1, lines 2476 and 2477 come into play, and those require an exact match.